### PR TITLE
Add default connection for redshift and update doc

### PR DESCRIPTION
**Pull request by vincbeck**
Add a default connection for Amazon Redshift. That should help users to set up connection to their Amazon Redshift clusters.
Also updated the connection documentation

### CODE REVIEW

File: airflow/utils/db.py [hunk 1]
```diff
@@ -508,6 +508,22 @@ def create_default_connections(session: Session = NEW_SESSION):
...
+            conn_id='redshift_default',
+            conn_type='redshift',
+            extra="""{
+    "iam": true,
+    "cluster_identifier": "my-redshift-cluster-identifier",
+    "port": 5439,
+    "profile": "default",
+    "db_user": "awsuser",
+    "database": "dev",
+    "region": "us-east-1"
```
Comment (Bowrna) [inline airflow/utils/db.py]: Is it ok to specify the region as the default parameter?

Comment (vincbeck) [reply airflow/utils/db.py]: It's a good question. It was to give an example but it also can be confusing and make the user think only us-east-1 is available, let me fix that

**Review by potiuk — APPROVED** [review]

File: airflow/utils/db.py (hunk 1 above)
Comment (potiuk) [reply airflow/utils/db.py]: Good point. I think thoguh leavin it empty is a better Idea ? What happens if someone uses the `<>` one?

**Review by potiuk — CHANGES_REQUESTED** [review]
The "<REDSHIFT_REGION>` might create some unexpected error? Can we leave it empty?

**Comment by vincbeck** [comment]
Done

**Review by potiuk — APPROVED** [review]

**Comment by vincbeck** [comment]
This PR is approved. Can we merge it?

**Review by potiuk — APPROVED** [review]

**Pull request merged**